app/presenters/account_show_presenter.rb

```diff
@@ -30,14 +41,49 @@ def show_service_provider_continue_partial?
     sp_name.present? && sp_session_request_url.present?
   end
 
-  def show_gpo_partial?
-    user.gpo_verification_pending_profile?
+  def showing_alerts?
+    show_service_provider_continue_partial? ||
+      show_password_reset_partial?
   end
 
-  def showing_any_partials?
-    show_service_provider_continue_partial? ||
-      show_password_reset_partial? ||
-      show_gpo_partial?
+  def active_profile?
+    return @active_profile if defined?(@active_profile)
+    @active_profile ||= user.active_profile.present?
+  end
+
+  def active_profile_for_authn_context?
+    return @active_profile_for_authn_context if defined?(@active_profile_for_authn_context)
+
+    @active_profile_for_authn_context = active_profile? && (
+      !authn_context.biometric_comparison? || identity_verified_with_biometric_comparison?
+    )
+  end
+
+  def pending_idv?
+    return @pending_idv if defined?(@pending_idv)
+    @pending_idv = authn_context.identity_proofing? && !active_profile_for_authn_context?
+  end
+
+  def pending_ipp?
+    return @pending_ipp if defined?(@pending_ipp)
+    @pending_ipp = user.pending_in_person_enrollment.present?
+  end
+
+  def pending_gpo?
+    return @pending_gpo if defined?(@pending_gpo)
+    @pending_gpo = user.gpo_verification_pending_profile?
+  end
```

Comment:
I think that the corresponding user methods are memoized here, maybe we could simplify all of these
```suggestion
  def pending_idv?
    authn_context.identity_proofing? && !active_profile_for_authn_context?
  end

  def pending_ipp?
    user.pending_in_person_enrollment.present?
  end

  def pending_gpo?
    user.gpo_verification_pending_profile?
  end
```